src/pages/new-claim/new-claim.tsx

```diff
@@ -86,7 +86,7 @@ export default function NewClaim() {
           <p className={globalStyles.bold}>Your claim ID is: {newClaimId}</p>
           <p className={styles.processMessage}>
             Your claim is being processed and will be voted on within 72 hours. Please check back for any updates and{' '}
-            <a href="https://docs.google.com" target="_blank" rel="noopener noreferrer">
+            <a href="https://docs.api3.org" target="_blank" rel="noopener noreferrer">
```

Comment:
The document will be served from the api3.org page right?

Reply:
The page doesn't exist yet, and it will be on docs.api3.org

Reply:
Yep I have a task already 👍